snapshot/restore/local_Fs_output_unix.go

```diff
@@ -0,0 +1,17 @@
+// +build linux darwin
```

Comment:
Should the file be named `local_fs_output_unix.go` ?

Reply:
thanks!